api/main.py

```diff
@@ -241,8 +247,15 @@ async def get_nodes_count(request: Request, kind: str = "node"):
                 status_code=status.HTTP_400_BAD_REQUEST,
                 detail=f"Invalid request parameters: {msg}"
             )
-    translated_params = model.translate_fields(query_params)
-    return await db.count(model, translated_params)
+
+    try:
+        translated_params = model.translate_fields(query_params)
```

Comment:
I've lost track of why this can't be done in `validate_params()`, but looking at the code here it doesn't seem right to be mixing the actual code handling logic and input validation logic.

Reply:
If it is by default raising `ValueError` for the conversion `datetime.fromisoformat`, why check it manually in `validate_params`?

Reply:
See my previous comment.